crates/bevy_render/src/render_resource/pipeline_specializer.rs

```diff
@@ -124,20 +125,19 @@ impl<S: SpecializedMeshPipeline> SpecializedMeshPipelines<S> {
                         }
                         err
                     })?;
+
                 // Different MeshVertexBufferLayouts can produce the same final VertexBufferLayout
                 // We want compatible vertex buffer layouts to use the same pipelines, so we must "deduplicate" them
+                let vertex_key = descriptor.vertex.buffers.get(0).cloned();
```

Comment:
Does this clone have any negative performance consequences?

Reply:
I don't see a good way to avoid it, but it should be a fairly small clone?

Reply:
To avoid the clone, I think we would have to wrap it in a Cow. Let me try that.

Reply:
Can't get it working, rip.